src/functions.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I was using the same technique the Composer autoloader class uses to scope isolate the loading of files to prevent access to class' `$this`/`self` [1].  Oh yeah, I meant to update `composer.json` to load our autoloader.  Nice catch.

[1] https://github.com/composer/composer/blob/master/src/Composer/Autoload/ClassLoader.php#L407